tests/e2e/specs/edit-posts.test.js

```diff
@@ -69,22 +73,24 @@ describe( 'Edit Posts', () => {
 		expect( editorPostTitleInput.length ).toBe( 1 );
 	} );
 
-	it( 'allows an existing post to be quick edited using the Quick Edit button', async () => {
+	test( 'allows an existing post to be quick edited using the Quick Edit button', async ( {
+		admin,
+		editor,
+		page
+	} ) => {
 		const title = 'Test Title';
-		await createNewPost( { title } );
-		await publishPost();
-		await visitAdminPage( '/edit.php' );
+		await admin.createNewPost( { title } );
+		await editor.publishPost();
+		await admin.visitAdminPage( '/edit.php' );
 
 		await page.waitForSelector( '#the-list .type-post' );
 
 		// Focus on the post title link.
-		const [ editLink ] = await page.$x(
-			`//a[contains(@class, "row-title")][contains(text(), "${ title }")]`
-		);
-		await editLink.focus();
+		await page.locator( `//a[contains(@class, "row-title")][contains(text(), "${ title }")]` ).focus();
 
 		// Tab to the Quick Edit button and press Enter to quick edit.
-		await pressKeyTimes( 'Tab', 2 );
+		await page.keyboard.press( 'Tab' );
+		await page.keyboard.press( 'Tab' );
```

Comment:
If we're using the playwright e2e utils package here, we can use the `pressKeys` util from the `PageUtils` fixture:

```js
await pageUtils.pressKeys( 'Tab', { times: 2 } )
```

Reply:
Ah I was looking for that!